gdx/src/com/badlogic/gdx/Input.java

```diff
@@ -742,6 +742,18 @@ private static void initializeKeyNames () {
 	/** @return whether the menu button is currently being caught */
 	public boolean isCatchMenuKey ();
 
+	/**

```

Comment:
I'm not a native speaker, but is it correct to say "catched"?
My browser is underlining it in red.

Reply:
I'm also not native, caught in the act. Will change all this incidences.